<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Conentration App</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="main_widget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>200</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="top_bar">
     <item>
      <widget class="QPushButton" name="btn_page_1">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:#8350AA;
}</string>
       </property>
       <property name="text">
        <string>Page 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_page_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:#8350AA;
}</string>
       </property>
       <property name="text">
        <string>Page 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_page_3">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:#8350AA;
}</string>
       </property>
       <property name="text">
        <string>Page 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_page_4">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:#8350AA;
}</string>
       </property>
       <property name="text">
        <string>Page 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>352</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Content">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>150</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_1">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>150</height>
         </size>
        </property>
        <widget class="QLabel" name="label_1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Page 1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>70</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-20</y>
           <width>351</width>
           <height>101</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>40</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Page 2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>110</y>
           <width>80</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>351</width>
           <height>151</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>40</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Page 3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>150</height>
         </size>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>351</width>
           <height>151</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>40</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Page 4</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
